examples/upgradeable-contracts/set-code-hash/updated-incrementer/Cargo.toml

```diff
@@ -10,7 +10,7 @@ publish = false
 [dependencies]
 ink_primitives = { path = "../../../../crates/primitives", default-features = false }
 ink_metadata = { path = "../../../../crates/metadata", default-features = false, features = ["derive"], optional = true }
-ink_env = { path = "../../../../crates/env", default-features = false, features = ["ink-debug"] }
```

Comment:
I think this feature removal is unintentional.

Reply:
I removed it to track the actual size of the contract